browser/scripts/checkbrowser.js

```diff
@@ -0,0 +1,45 @@
+checkBrowser = function() {
```

Comment:
so if something from E2.app fails in unsupported browser - user will never see the message. that's why it should be called before E2 initialization